### ISSUE TITLE
Estimate fees

### ISSUE THREAD

**Issue by jaonoctus** [brunoerg/genetic-bitcoin#1]
The current algorithm works just fine, given a fixed fee. But we know that the fee will change based on the outputs length.

**Comment by brunoerg** [comment]
Yeah! We can cansider how much the user wanna pay in sats/B. However, to consider it, we have to modify our simulation to consider not only the UTXO value but also its size. Right?
